### ISSUE TITLE
Storage Investment // Existing Capacity for the flows not included

### ISSUE THREAD

**Issue by FabianTU** [oemof/oemof#503]
Hello all,

I just ran into the issue, that my model invested into a storage and maxed everything out (capacity Investment and flow Investment). Now I took those results and threw them into the model again (iterative) and the model went infeasible.
Reason/Example: The model invested into a PHES capacity of 42 000 Units and also into flows of 7000 each (that is the Maximum for the flows I allowed and thus additional Investment equals 0) - when I then rerun the model with the old values, the constraint demands: 
expr = invest_flow == (42 000 + invest_capacity)* Ratio. Since the left side of the equation has to be 0 and the right Hand side is always greater than Zero - infeasible.

```python

def _storage_capacity_inflow_invest_rule(block, n):

            expr = (m.InvestmentFlow.invest[i[n], n] ==

                    (n.investment.existing + self.invest[n]) *

                    n.invest_relation_input_capacity)

            return expr

        self.storage_capacity_inflow = Constraint(

            self.INVEST_REL_CAP_IN, rule=_storage_capacity_inflow_invest_rule)
```
Now, what appoach is more Fitting, adding the existing flow capacity into the equation or delete the existing capacity from the right Hand side of the equation?

I think both approaches might be reasonable, it depends on what I want to do.
If we delete the `n.investment.existing` ONLY the NEW Investment will be bounded by the constraint. I think that would be easier to work with if I want to add existing capacity etc.

What do you think? Is it an issue after all or am I mistaken and use the model incorrectly?

Best
Fabian

**Comment by uvchik** [comment]
But anyway I will fix this bug. See #504.

**Comment by uvchik** [comment]
Fixed with #504.
